ironfish-zkp/src/lib.rs

```diff
@@ -7,5 +7,6 @@ pub use zcash_primitives::sapling::{
 };
 
 pub mod proofs {
-    pub use zcash_proofs::circuit::sapling::{Output, Spend};
+    pub use crate::circuits::{output::Output, spend::Spend};
+    pub use crate::constants::PUBLIC_KEY_GENERATOR;
```

Comment:
This is already accessible from `ironfish_zkp::constants::PUBLIC_KEY_GENERATOR`, so we don't need to duplicate the export here.
```suggestion
```